cmd/serverless-init/initcontainer/initcontainer_test.go

```diff
@@ -63,3 +68,35 @@ func TestFlushTimeout(t *testing.T) {
 	assert.Equal(t, false, metricAgent.hasBeenCalled)
 	assert.Equal(t, false, mockLogsAgent.DidFlush())
 }
+
+func TestPropagateChildSuccess(t *testing.T) {
+	err := execute(&serverlessLog.Config{}, []string{"bash", "-c", "exit 0"})
```

Comment:
It wouldn't be a bad idea to include a build constraint to only Linux, something like `// +build linux`, to avoid this test failing if we're in an environment where we don't have bash.

Reply:
oh thanks, I didn't think about it!